streams/streams-scala/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,35 @@
+## /*
```

Comment:
Markup seems weird? Why do you have JavaDoc comment markup?

Would a single `#` not be sufficient?

Reply:
done ..